### DISCUSSION TITLE
Is `F` mode linear or sRGB piecewise gamma?

### DISCUSSION THREAD

**Discussion by elias-gaeros** [python-pillow/Pillow#8452] [Q&A] [answered]
Floating point pixels are usually linear light, but with Pillows "experimental modes" who knows?
Also, what about `I;16`?

**Comment by radarhere** [comment, marked as answer]
https://github.com/python-pillow/Pillow/issues/1604#issuecomment-166347494
> There is essentially no color space awareness in Pillow by default.

If you are interested in color spaces though, you can look at https://pillow.readthedocs.io/en/stable/reference/ImageCms.html

**Comment by elias-gaeros** [comment]
Ok, thanks.

This discussion about processing images in linear 8-bits RGB because Pillow can't do any better shows that it shouldn't be used for anything beside image I/O.
